integration-tests/api/__tests__/admin/__snapshots__/price-list.js.snap

```diff
@@ -21,7 +21,6 @@ Object {
       "price_list_id": "pl_no_customer_groups",
       "region_id": null,
       "updated_at": Any<String>,
-      "variant_id": "test-variant",
```

Comment:
suggestion: With the latest changes around the hooks, shouldn't we have the variant_id present? Maybe we are missing some relations now that they are detached `*.variants` to load it. Otherwise, we are including breaking changes in the current API. for the profiles, this is what I've done to maintain the backward compat.